modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -144,14 +171,26 @@ public ListTypeEntry updateListTypeEntry(
 		ListTypeEntry listTypeEntry = listTypeEntryPersistence.findByPrimaryKey(
 			listTypeEntryId);
 
+		_validateName(nameMap);
+
+		listTypeEntry.setNameMap(nameMap);
+
+		ListTypeDefinition listTypeDefinition =
+			_listTypeDefinitionPersistence.findByPrimaryKey(
+				listTypeEntry.getListTypeDefinitionId());
+
+		if (listTypeDefinition.isSystem() &&
```

Comment:
I didn't understand this logic, is it correct? It's updating listTypeEntry when satisfied.
Maybe we should call `_validateInvokerBundle` here.

Reply:
@guilhermedcamacho,
We need to do this, because in the case of the picklist we're going to let the user change at least the label.